src/Update.hs

```diff
@@ -273,7 +274,7 @@ publishPackage log updateEnv oldSrcUrl newSrcUrl attrPath result opDiff msgs = d
         if u == T.empty
           then ""
           else "\n\nmeta.homepage for " <> attrPath <> " is: " <> u
-  let rewriteMessages = foldl (\ms m -> ms <> T.pack "\n- " <> m) "" msgs
+  let rewriteMessages = foldl (\ms m -> ms <> T.pack "\n- " <> m) "Updates performed:" msgs
```

Comment:
This adds a 1-liner to the stock version upgrade, so this section will always exist, and then gives it a header.